packages/odyssey-react-mui/src/themes/odyssey/components.ts

```diff
@@ -18,24 +18,57 @@ export const components: ThemeOptions["components"] = {
     styleOverrides: {
       root: ({ ownerState, theme }) => ({
         padding: theme.spacing(4),
+        gap: theme.spacing(4),
         color: theme.palette.text.primary,
         ...(ownerState.severity && {
           backgroundColor: theme.palette[ownerState.severity].lighter,
           borderColor: theme.palette[ownerState.severity].light,
         }),
+        ...(ownerState.variant === "banner" && {
+          position: "relative",
+          justifyContent: "center",
+          alignItems: "center",
+          borderWidth: 0,
+        }),
         ...(ownerState.variant === "infobox" && {
           borderStyle: "solid",
           borderWidth: 1,
           "&:not(:last-child)": {
             marginBottom: theme.spacing(4),
           },
         }),
+        ...(ownerState.variant === "toast" && {
+          maxWidth: theme.mixins.maxWidth,
+          borderStyle: "solid",
+          borderWidth: 1,
+          position: "relative",
+          alignItems: "start",
+        }),
+      }),
+      action: ({ ownerState, theme }) => ({
         ...(ownerState.variant === "banner" && {
-          borderWidth: 0,
+          padding: 0,
+          marginRight: 0,
+          top: "50%",
+          right: theme.spacing(4),
+          position: "absolute",
+          transform: "translateY(-50%)",
+        }),
+        ...(ownerState.variant === "toast" && {
+          position: "absolute",
+          top: `calc(${theme.spacing(4)} - ${theme.spacing(1)} + ${
```

Comment:
This accounts for the calculated size of `IconButton`. Once we theme `IconButton` to match our spec, we can redo the layout here to be more generalized.